src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -437,20 +451,32 @@ export default class rd2EntryForm extends LightningElement {
     * @param paymentMethod Payment method
     */
     evaluateElevateWidget() {
-        const isOpenSchedule = (this.scheduleComponent && this.scheduleComponent.getRecurringType() === RECURRING_TYPE_OPEN);
+        const isScheduleSupported = this.isScheduleSupported();
         const isValidPaymentMethod = this.isElevatePaymentMethod();
         const currencySupported = this.isCurrencySupported();
         const countrySupported = this.isCountrySupported();
         this.isElevateWidgetEnabled = this.isElevateEditWidgetEnabled
             || (this.isElevateCustomer === true
             && isValidPaymentMethod
-            && isOpenSchedule
+            && isScheduleSupported
             && currencySupported
             && countrySupported);
 
         this.populateCardHolderName();
     }
 
+    isScheduleSupported() {
+        if(this.scheduleComponent) {
+            const isValidRecurringType = this.scheduleComponent.getRecurringType() === RECURRING_TYPE_OPEN;
```

Comment:
`getRecurringType` is returning `null` on Edit of an existing Elevate RD, so the Credit Card widget doesn't show up 😬

Reply:
I added a comment to `evaluateElevateEditWidget` from a previous issue, I wound up using getFieldValue to compare on load instead of calling out to the scheduleComponent.